context/app/static/js/shared-styles/charts/VerticalStackedBarChart/VerticalStackedBarChart.stories.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { scaleLinear, scaleOrdinal, scaleBand } from '@visx/scale';
+
+import VerticalStackedBarChartComponent from './VerticalStackedBarChart';
+
+export default {
+  title: 'Charts/VerticalStackedBarChart',
+  component: VerticalStackedBarChartComponent,
+};
+
+export const VerticalStackedBarChart = (args) => <VerticalStackedBarChartComponent {...args} />;
+const yScale = scaleLinear({
+  domain: [0, 13],
+  nice: true,
+});
+
+const xScale = scaleBand({
+  domain: [1, 2, 3, 4, 5],
+  padding: 0.2,
+});
+
+const colorScale = scaleOrdinal({
+  domain: ['matched', 'unmatched'],
+  range: ['#DA348A', '#6C8938'],
+});
+
+VerticalStackedBarChart.args = {
+  parentWidth: 500,
+  parentHeight: 500,
+  visxData: [
+    {
+      cluster: 1,
```

Comment:
```suggestion
      cluster: 'cluster A',
```
These are just the labels on the axis, right? I'd suggest strings in the demo.